<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Автоматизована система розпізнавання емоцій</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox_emotion_portret">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>271</width>
      <height>291</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="title">
     <string>Емоційний портрет:</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="1">
      <widget class="QProgressBar" name="progressBar_disgusted">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_disgusted">
       <property name="text">
        <string>Відраза:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_happines">
       <property name="text">
        <string>Щастя:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_surprised">
       <property name="text">
        <string>Здивування:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QProgressBar" name="progressBar_surprized">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QProgressBar" name="progressBar_sad">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QProgressBar" name="progressBar_happiness">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_sadness">
       <property name="text">
        <string>Сум:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_anger">
       <property name="text">
        <string>Злість:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QProgressBar" name="progressBar_anger">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_tools">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>271</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>Інструменти:</string>
    </property>
    <widget class="QToolBox" name="toolBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>231</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_teach">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>177</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Тренування</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_file_path">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>201</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Вчити</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>140</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Відмінити</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_skip">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Пропустити</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Добре</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stop">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>81</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Зупинити</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_by_images">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>0</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>За зображеннями</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_by_features">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>За зробленими вимірами</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_test">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>251</width>
        <height>177</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Тестування</string>
      </attribute>
      <widget class="QPushButton" name="pushButton_test">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>211</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Отримати емоційний портрет</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_image_open">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>211</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Відкрити фото...</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_View">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>631</width>
      <height>551</height>
     </rect>
    </property>
    <property name="title">
     <string>Зображення обличчя:</string>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>14</y>
       <width>611</width>
       <height>531</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="action_manual">
   <property name="text">
    <string>Інструкція..</string>
   </property>
  </action>
  <action name="action_support">
   <property name="text">
    <string>Технічна підтримка..</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>Про програму..</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Про розробника..</string>
   </property>
  </action>
  <action name="action_set_ua">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Українська</string>
   </property>
  </action>
  <action name="action_set_ru">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="action_set_en">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Вихід</string>
   </property>
  </action>
  <action name="action_authentify">
   <property name="text">
    <string>Авторизація..</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
